draft-ietf-scim-device-model.mkd

```diff
@@ -381,9 +381,11 @@ If present, this field contains a dnsName, as specified in Section
 necessarily bind to the incoming IP address of the application client.
 This attribute is not required, read write, singular and NOT case
 sensitive.  This name may also appear as an subjectAlternateName (SAN)
-of type dnsName, in which case the subject MUST be ignored.  If
-multiple dnsNames are present, it is left to server implementations to
-address any authorization conflicts associated with those names.
+of type dnsName, in which case the subject MUST be ignored and the
```

Comment:
> in which case the subject MUST be ignored
Here, subject refers to the subject in the certificate and not the `subjectName` attribute this paragraph is talking about right?

Reply:
subjectName

Reply:
I am confused. 
> This name may also appear as an subjectAlternateName (SAN)
> of type dnsName, in which case the subject MUST be ignored

_"This name"_ refers to the `subjectName` attribute right?

Reply:
Yes